diff.py

```diff
@@ -1899,6 +1947,35 @@ def process(dump: str, config: Config) -> List[Line]:
     return output
 
 
+# Post-process the asm lines in-place. Includes:
+#  - De-referencing relocations
+def post_process(lines: List[Line]):
+    post_process_relocations(lines)
+
+def post_process_relocations(lines: List[Line]):
```

Comment:
This seems specific to ARM32, should it be part of the ARM32 config? (or DifferenceNormalizer)

Reply:
I created a PostProcessor class and moved this logic under PostProcessorARM32. Please let me know if you have any concerns with this.